### ISSUE TITLE
Remodel throwing error when writing the place file

### ISSUE THREAD

**Issue by j-meiring** [rojo-rbx/remodel#93]
Hi all,

We have a fully managed Rojo project, which we have successfully implemented in conjuction with Remodel. I've written a `PullAssets.lua` script that successfully remodels our `.rbxm` files, on two out of three machines.

Working: MacBook Pro, MacOs Monterey 12.4
Working: Windows PC, Windows 10
**Not working**: MacBook Pro, MacOS Monterey 12.6

The only difference that I can see is that the Mac that isn't working has been updated while mine has not. I have set up that machine to use the exact environment that I have set up on my Mac, but when running `remodel run ./scripts/PullAssets.lua`, the console throws the following error:

```
[ERROR] callback error
  stack traceback:
    [C]: in field 'writePlaceFile'
    [string "PullAssets.lua"]:69: in local 'saveAssetsToGame'
    [string "PullAssets.lua"]:72: in main chunk
    caused by: error converting Lua userdata to String (expected string or number)
```

Here is a snippet where you can see more or less what that code looks like (screenshot so you can see corresponding line numbers):

![Screenshot 2022-11-23 at 18 03 33](https://user-images.githubusercontent.com/24519810/203593187-3c68bcd4-0ccf-444e-bf29-460af444eb1f.png)

Would you be able to let me know if this is a known issue, or if it may be something raised by the release of Monterey 12.6?
If there is something that I can do in my script, could you please point me in the right direction?

Thanks in advance! Please let me know if you need any further information.

**Comment by eAi** [comment]
@j-meiring Make sure all your computers are running the same, latest remodel version. Recently the order or parameters for `writePlaceFile` (and `writeModelFile`) were changed around (annoyingly - it's caused us lots of similar issues!)

**Comment by j-meiring** [comment]
@eAi thanks for the response. We did think of that, so I actually checked the versions - we are both using [v0.11.0](https://github.com/rojo-rbx/remodel/releases/tag/v0.11.0).

**Comment by eAi** [comment]
The error seems to be caused by `writePlaceFile` getting userData where it expects a string. I believe the 'game' argument to writePlaceFile is userData, so it seems really very likely that the issue is the parameter order. 

How have you checked your remodel version - with `remodel --version`?

I'm running Monterey 12.6.1 and this works fine for me, I tried this minimal test case - you could try the same:
```
local game = remodel.readPlaceFile("Place.rbxlx")
remodel.writePlaceFile("Test.rbxlx", game)
```

**Comment by j-meiring** [comment]
Thanks again, so we did another check, and it seems that since I updated the environment to reflect mine (which is definitely 11.0.0), there may have been some caching in the terminal that the path was still accessing the old outdated version of the script. Turns out I checked the version on mine, but not in the terminal when we tested it on his machine.

After checking it in a fresh terminal, it's now working! I feel a bit silly, we went through a troubleshooting session with the exact thinking that it must be the version, and it was a caching issue and we had actually resolved it.

Thanks so much for taking the time to help out, we really appreciate it. You've been a great help.
